docs/data/tree-view/simple-tree-view/selection/SingleSelectTreeView.tsx

```diff
@@ -1,42 +1,28 @@
 import * as React from 'react';
 import Box from '@mui/material/Box';
-import { RichTreeView } from '@mui/x-tree-view/RichTreeView';
-import { TreeViewBaseItem } from '@mui/x-tree-view/models';
-
-const MUI_X_PRODUCTS: TreeViewBaseItem[] = [
-  {
-    id: 'grid',
-    label: 'Data Grid',
-    children: [
-      { id: 'grid-community', label: '@mui/x-data-grid' },
-      { id: 'grid-pro', label: '@mui/x-data-grid-pro' },
-      { id: 'grid-premium', label: '@mui/x-data-grid-premium' },
-    ],
-  },
-  {
-    id: 'pickers',
-    label: 'Date and Time Pickers',
-    children: [
-      { id: 'pickers-community', label: '@mui/x-date-pickers' },
-      { id: 'pickers-pro', label: '@mui/x-date-pickers-pro' },
-    ],
-  },
-  {
-    id: 'charts',
-    label: 'Charts',
-    children: [{ id: 'charts-community', label: '@mui/x-charts' }],
-  },
-  {
-    id: 'tree-view',
-    label: 'Tree View',
-    children: [{ id: 'tree-view-community', label: '@mui/x-tree-view' }],
-  },
-];
+import { SimpleTreeView } from '@mui/x-tree-view/SimpleTreeView';
+import { TreeItem } from '@mui/x-tree-view/TreeItem';
 
 export default function SingleSelectTreeView() {
   return (
-    <Box sx={{ minHeight: 200, flexGrow: 1, maxWidth: 400 }}>
-      <RichTreeView items={MUI_X_PRODUCTS} />
+    <Box sx={{ minHeight: 352, minWidth: 250 }}>
```

Comment:
With the new height, going to https://mui.com/x/react-tree-view/simple-tree-view/selection/ and scrolling the page down to read the content doesn't feel as great as it could be. There are a lot of empty spaces. 

Was "Fix the layout shift on all demo when you expand all the items" a problem in the first place?

```suggestion
    <Box sx={{ minHeight: 200, minWidth: 250 }}>
```